Library/Formula/swagger-codegen.rb

```diff
@@ -0,0 +1,23 @@
+class SwaggerCodegen < Formula
+  desc "Generation of client and server from Swagger definition"
+  homepage "http://swagger.io/swagger-codegen/"
+  url "https://github.com/swagger-api/swagger-codegen/archive/v2.1.4.tar.gz"
+  sha256 "b4965158fb8abbebdfcaab2fdc0c5556001c587a526279d44ea9878f7ef9ef4c"
+
+  head do
+    url "https://github.com/swagger-api/swagger-codegen.git"
+  end
+
+  depends_on :java => "1.7+"
+  depends_on "maven" => :build
+
+  def install
+    system "mvn", "clean", "package"
+    libexec.install "modules/swagger-codegen-cli/target/swagger-codegen-cli.jar"
+    bin.write_jar_script libexec/"swagger-codegen-cli.jar", name
+  end
+
+  test do
+    system "#{name} help generate | grep 'Generate code with chosen lang'"
```

Comment:
Can’t we write a test that does something more substantial than e.g. `--help`/`--version`?